Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -109,10 +96,28 @@ private extension ReceiptRenderer {
                         <h3>\(Localization.summarySectionTitle.uppercased())</h3>
                         \(summaryTable())
                     </p>
+                    <p>
+                        \(regulatoryItems())
+                    </p>
                 </body>
             </html>
         """
     }
+}
+
+
+private extension ReceiptRenderer {
+    private func configureHeaderAndFooter() {
+        headerHeight = Constants.headerHeight
+        footerHeight = Constants.footerHeight
+    }
+
+    private func configureFormatter() {
+        let formatter = UIMarkupTextPrintFormatter(markupText: htmlContent())
+        formatter.perPageContentInsets = .init(top: Constants.headerHeight, left: Constants.marging, bottom: Constants.footerHeight, right: Constants.marging)
+
+        addPrintFormatter(formatter, startingAtPageAt: 0)
+    }
```

Comment:
I didn't catch it before, but same question for line.title : how do we avoid tags or brackets breaking the generated html?

Reply:
We don't, I missed this. I've logged #4137 for it